utils/documentation_tests.txt

```diff
@@ -35,6 +35,7 @@ src/transformers/models/marian/modeling_marian.py
 src/transformers/models/mbart/modeling_mbart.py
 src/transformers/models/mobilebert/modeling_mobilebert.py
 src/transformers/models/mobilebert/modeling_tf_mobilebert.py
+src/transformers/models/opt/modeling_opt.py
```

Comment:
@ArthurZucker we should always try to test our docstrings to be sure they work when people look into the docs :-)

Reply:
Oh that's where you put it !